package chapter.eight.exercise;

public class Ex8_2 {

  /*
  * sol:
  * ① : ㅇ → 에러 메세지를 통해 해당 에러가 발생한 시점의 호출 스택을 알 수 있다.
  * ② : ㅇ
  * ③ : ㅇ → 에러 메세지를 통해 발생한 예외의 이름과 원인을 알 수 있다.
  * ④ : X → 호출 스택을 살펴보면 method2 메서드가 method1 을 호출한 것이 아니고 main, method1(), method2() 순으로 메서드가 호출되었음을 알 수 있다.
  *
  *
  *
  * */

}
